cmd/zarb/start.go

```diff
@@ -84,6 +90,19 @@ func Start() func(c *cli.Cmd) {
 				return
 			}
 
+			// separate pprof handlers from DefaultServeMux.
+			pprofMux := http.DefaultServeMux
+			http.DefaultServeMux = http.NewServeMux()
+			if pprofOpt != nil && *pprofOpt != "" {
```

Comment:
`pprofOpt != nil` As I checked `mow.cli` option will never be null.

Reply:
@b00f  in that case it's redundant